conf/ocsci/ocp-4.3-upgrade.yaml

```
@@ -0,0 +1,5 @@
+---
+# Use this config for upgrading of OCP 4.3 cluster
+UPGRADE:
+  # ocp_upgrade_version: "4.3.0-0.nightly"
```

This one should represent the latest nightly right?

So I think that you will have to have this uncommented 
```suggestion
  ocp_upgrade_version: "4.3.0-0.nightly"
```